VideoWeb/VideoWeb/ClientApp/angular.json

```diff
@@ -1,192 +1,178 @@
-                            "src/app/services/clients/**",
-                            "src/app/vh-officer/helper.ts",
-                            "src/app/services/audio-only-image.service.ts"
-                        ],
-                        "styles": [
-                            "src/styles.scss"
-                        ],
-                        "stylePreprocessorOptions": {
-                            "includePaths": [
-                                "src/sass"
-                            ]
-                        },
-                        "scripts": [
-                            "src/scripts/vodafone/pexrtc.js",
-                            "src/scripts/EdgePF.js",
-                            "src/scripts/IEPF.js",
-                            "src/scripts/Heartbeat.js",
-                            "node_modules/chart.js/dist/chart.umd.js"
-                        ],
-                        "assets": [
-                            "src/assets",
-                            {
-                                "glob": "*",
-                                "input": "node_modules/govuk-frontend/govuk/assets/fonts",
-                                "output": "assets/fonts/"
-                            }
-                        ]
-                    }
-                },
-                "server": {
-                    "builder": "@angular-devkit/build-angular:server",
-                    "options": {
-                        "outputPath": "dist-server",
-                        "main": "src/main.ts",
-                        "tsConfig": "src/tsconfig.server.json",
-                        "sourceMap": true,
-                        "optimization": false,
-                        "buildOptimizer": false
-                    },
-                    "configurations": {
-                        "dev": {
-                            "optimization": true,
-                            "outputHashing": "all",
-                            "namedChunks": false,
-                            "extractLicenses": true,
-                            "buildOptimizer": true
-                        },
-                        "production": {
-                            "optimization": true,
-                            "outputHashing": "all",
-                            "sourceMap": false,
-                            "namedChunks": false,
-                            "extractLicenses": true,
-                            "buildOptimizer": true
-                        }
-                    },
-                    "defaultConfiguration": ""
-                },
-                "lint": {
-                    "builder": "@angular-eslint/builder:lint",
-                    "options": {
-                        "lintFilePatterns": [
-                            "src/**/*.ts",
-                            "src/**/*.html"
-                        ]
-                    }
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
are there any changes needed to this file? It's a bit tricky to tell with the changes to the formatting